kong/dao/schemas_validation.lua

```diff
@@ -185,7 +195,7 @@ function _M.validate_entity(tbl, schema, options)
               end
             end
 
-            if t[column] and type(t[column]) == "table" then
+            if t[column] and t[column] ~= ngx.null and type(t[column]) == "table" then
```

Comment:
Since `type(ngx.null) == "userdata"`, it seems we can stick to `if t[column] and type(t[column]) == "table" then` here? Explicitness is good though. Maybe a comment saying `ngx.null` will also get skipped?